src/main/java/com/almanza/inventory/service/impl/ShoeServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package com.almanza.inventory.service.impl;
+
+import com.almanza.inventory.models.ShoeModel;
+import com.almanza.inventory.models.ShoeToAddModel;
+import com.almanza.inventory.models.ShoeToUpdateModel;
+import com.almanza.inventory.service.ShoeService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ShoeServiceImpl implements ShoeService {
+
+    @Override
+    public ResponseEntity<List<ShoeModel>> getShoes() {
+        return ResponseEntity.status(HttpStatus.OK).body(
+            List.of(
```

Comment:
just return the list and move the code related to Rest to the controller (Communication layer)